main.go

```diff
@@ -13,25 +13,25 @@ import (
 func main() {
 	config.Parse()
 	appConf := config.GetConfig()
+	s := appConf.SQLConf()
+	// migration
+	h, err := infrastructure.NewSQLHandler(s)
+	if err != nil {
+		log.Fatal(err)
+	}
 
 	if appConf.IsMigrate() {
-		s := appConf.SQLConf()
-		h, err := infrastructure.NewSQLHandler(s)
-		if err != nil {
-			log.Fatal(err)
-		}
-
-		if appConf.IsDevelopment() {
-			if err := mockdata.InsertSampleDataToDB(h); err != nil {
-				log.Fatal(err)
-			}
-		}
-
 		log.Println("finished")
```

Comment:
差分外ですがついでに:pray:
英語自信ないので適当に修正してもらえると助かります
```suggestion
		log.Println("a migration is finished")
```